lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
+    '`crypto.randomBytes(size)` is deprecated. ' +
+    'Please use `crypto.randomBytesSync(size)` instead.');
+exports.randomBytesSync = randomBytes;
+exports.pseudoRandomBytes = util.deprecate(randomBytes,
+    '`crypto.pseudoRandomBytes()` is deprecated. ' +
+    'Please use `crypto.randomBytes()` instead.');
```

Comment:
These general need a general rewrite now. I think it should be possible to drop the sync deprecate addition altogether. I'll see if I come up with suitable replacements here.